smart-contracts/contracts/dex-router-osmosis/README.md

```diff
@@ -0,0 +1,9 @@
+# Dex router for osmosis
+
+Responsible for the on-chain computation of multi-hop swap routes.
+Currently only supports pools with 2 tokens.
+
+TODO:
+ * Fix queries for best path
+ * Add checks and unit-tests
+ * Add support for (optional) on-chain detection of the optimal swap path
```

Comment:
Can add a todo for test-tube tests.

Reply:
Thx, actually that list wasn't updated properly by me. Fixed it.
Now, the only item is "possibly test-tube tests". Wrote possibly as I have some concerns that that might be more work than we get in return from it.